### ISSUE TITLE
Remove CAPTCHA

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#220]
CAPTCHA doesn't meet accessibility requirements (although there is a direct email address as an alternative to the form)

http://data.gov/contact/

**Comment by kvuppala** [comment]
this is complete.
